### DISCUSSION TITLE
Webcam Sharing

### DISCUSSION THREAD

**Discussion by Shreyanshmmgn** [mozilla/hubs#2735] [General Questions & Discussion] [answered]
Does sharing our webcam from different laptops/webcam/phone spawns different size screens inside the environment ?? Because when I and my friend are sharing our webcams inside Hubs our size of spawned webcams is different. So can we change the size of this!!

**Comment by emclaren** [comment]
I'm not sure about webcams specifically, but I know when people share emoji's inside of Hubs, the style depends on the type of device they are using, so I imagine it depends on the output resolution of the webcam. You can change their size by pointing at the screen and opening the secondary menu (press space if you are on desktop), and there should be a button with a few arrows pointing outwards that should change the size (see screencapture on a fox object)
![intro-hubs-object-menu-min2](https://user-images.githubusercontent.com/6835324/88820682-79523c00-d176-11ea-8d0a-6f2421a7b652.jpeg)

**Comment by InfiniteLee** [comment, marked as answer]
@Shreyanshmmgn Currently, all shared webcam/desktop screens are "scaled" the same but may differ in dimensions due to different camera/display resolution and aspect ratios. It's possible that this is the problem you are experiencing, but if you think it is something else, please provide a screenshot so we can better understand your issue.

**Reply by Shreyanshmmgn** [reply]
Yep, this is the problem that I am facing. So do we have any way of making their aspect ratios or resolution same?

**Reply by InfiniteLee** [reply]
This is not a feature that is exposed on Hubs atm. If you are running your own Hubs Cloud instance, you can possibly modify the constraints used with getUserMedia to adjust the camera resolution:
https://github.com/mozilla/hubs/blob/6fed1b39640197c57e2070153966424c3aff832c/src/scene-entry-manager.js#L442
However, I'm not sure that this will actually let you change the aspect ratio of your camera, since that is defined by your hardware. It may be possible, if your webcam supports it, to change the aspect ratio/resolution that it outputs, however that will be platform/device dependent.
